msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md

```diff
@@ -22,6 +22,9 @@ The search command is invoked from any one or both of the following locations:
 * Compose message area: The buttons at the bottom of the compose message area.
 * Command box: By @mentioning in the command box.
 
+   > [!NOTE]
+   > @mention is no longer supported for messaging extension.
```

Comment:
Can we change the note to "@mentioning message extensions is no longer supported in the compose box."